packages/uix-core/src/types.ts

```diff
@@ -219,4 +253,11 @@ export interface Emits<Events extends NamedEvent = NamedEvent>
     type: Type,
     listener: (ev: Extract<Events, { type: Type }>) => unknown
   ): () => void;
+  /**
+   * Same as EventTarget.removeEventListener but typed.
+   */
+  removeEventListener<Type extends Events["type"]>(
+    type: Type,
+    listener: (ev: Extract<Events, { type: Type }>) => unknown
+  ): void;
```

Comment:
I didn't notice that `removeEventListener` was missing in this interface until I tried to call it from code.